src/main/resources/db/migration/V12__add_policy_template_table.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE policy_templates (
+    name VARCHAR(512) PRIMARY KEY,
+    status VARCHAR(512) NOT NULL, -- ACTIVE, ARCHIVED
+    is_root_policy BOOLEAN NOT NULL,
+    statements text NOT NULL,
+
+    created_at timestamp NOT NULL,
+    updated_at timestamp NOT NULL
+);
+
+
+
+INSERT INTO policy_templates values (
```

Comment:
Why do we have a policy template table ? Do we foresee many pre-defined templates created ? Or do we support customer creating policy templates in future ?

We could have kept the templates as resource files as well in code.

Reply:
Also, Once we get an Admin console for IAM, having a table will be helpful